man/systemd.exec.xml

```diff
@@ -1261,6 +1261,14 @@
         filter is reset, all prior assignments will have no effect. This does not affect commands prefixed with
         <literal>+</literal>.</para>
 
+        <para>Note that strict system call filters may impact execution and error handling code paths of the service
+        invocation. Specifically, access to the <function>execve</function> system call is required for the execution
+        of the service binary – if it is blocked service invocation will necessarily fail. Also, if execution of the
```

Comment:
hmm, how do i type that on my german X11 kbd? I can type these three characters easily here:

- (plain "-" key)
– ("-" key combined with AltGr)
— ("-" key combined with Shift-AltGr)

Which one is the right one in your opinion?